src/components/heading/heading.module.css

```diff
@@ -0,0 +1,41 @@
+/*
+size prop classes
+*/
+.display-100 {
```

Comment:
@didoo When you say "directly use the hds helper classes in the component" do mean something like the following?

```tsx
const Heading: React.FC<HeadingProps> = ({
  level,
  size,
  slug,
  weight,
  ...rest
}) => {
  const id = slug
  const className = classNames(
    // before: s[`display-${size}`],
    `hds-typography-display-${size}`,
    // before: s[`weight-${weight}`],
    `hds-font-weight-${weight}`,
    rest.className
  )
  const passableProps = {
    ...rest,
    className,
    id,
  }

  const HeadingElement = `h${level}` as React.ElementType
  return <HeadingElement {...passableProps} />
}
```

Reply:
Yes, exactly. I am not suggesting to do it, but one may argue "why not" and I am curious to understand the thinking behind these "extra" classes. As I said, I can imagine is to decouple the classes from the design system from the classes used internally in JS, and since you're using CSS Modules, this makes it neat (and possible). Is this the reason? or there are others that I don't know/see?

Reply:
I think applying the classes in the component in this case is better. Having abstractions on top of the `hds` classes is helpful if we need to set extra CSS properties within them, but we don't need that yet and I'm not sure if / when we will. Applying the `hds` classes in the component like you're suggesting makes the `heading.module.css` file unnecessary, so I'm all for that simplicity! I will making this change here and also in the `Paragraph` component. Thank you for asking this question! 😊

Reply:
OK, good that we found a way to simplify things 👍